.envrc

```diff
@@ -0,0 +1,2 @@
+use flake
```

Comment:
I'd avoid including this change

Reply:
Should I add this file to `.gitignore` in the #194 PR and rebase this PR?